Core/MIPS/IR/IRCompVFPU.cpp

```diff
@@ -1179,10 +1227,12 @@ namespace MIPSComp {
 				}
 			}
 			return;
-		} else if (msz == M_4x4 && !IsConsecutive4(sregs)) {
+		} else if (msz == M_4x4 && IsConsecutive4(sregs)) {
+			// Consecutive, which is harder.
```

Comment:
I wonder if we could transpose into all 16 temp regs, and then use the transposed algorithm?

We can do dots if t and s are consecutive, but not great for NEON (only good for SSSE3, where I guess we'd mask and sum?)

We can use VLD4 / LD4 to transpose on ARM, so it can be pretty efficient if they're not already loaded.

-[Unknown]